IMDA-BTG/aiverify
# Fix/gitignore

## Description

1) Update the top level .gitignore to remove excessive patterns. Individual projects should have their own .gitignore specific to the project.

2) Add .gitignore  under stock-plugins folder for plugin specific gitignore.

## Motivation and Context

The current patterns in the top-level .gitignore has resulted in files, that are meant to be committed, to be ignored. It is better to have each project maintain .gitignore specific to the projects, and let the top-level .gitignore ignore build and test artifacts, as well as the uploads directory.

## Type of Change













## How to Test

Check out the branch and ensure that files and folders that are meant to be ignored, are being ignored. Files and folders that should not be ignore are NOT being ignored.

## Checklist

Please check all the boxes that apply to this pull request using "x":


## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

[Add any additional information or context that might be relevant to reviewers.]
